inlong-agent/agent-installer/src/main/java/org/apache/inlong/agent/installer/ModuleManager.java

```diff
@@ -213,6 +219,49 @@ private boolean updateModules(List<ModuleConfig> managerModuleList) {
         return true;
     }
 
+    private boolean downloadModule(ModuleConfig module) {
+        LOGGER.info("download module {} begin", module.getId());
+        try {
+            LOGGER.info("download url {}", module.getPackageConfig().getDownloadUrl());
+            URL url = new URL(module.getPackageConfig().getDownloadUrl());
+            URLConnection conn = url.openConnection();
+            Map<String, String> authHeader = httpManager.getAuthHeader();
+            authHeader.forEach((k, v) -> {
+                conn.setRequestProperty(k, v);
+            });
+            InputStream inStream = conn.getInputStream();
+            String path = module.getPackageConfig().getStoragePath() + "/" + module.getPackageConfig().getFileName();
+            FileOutputStream fs = new FileOutputStream(path);
```

Comment:
resource leak for the InputStream and FileOutputStream

Reply:
ok